http://sourceware.org/bugzilla/show_bug.cgi?id=12354

fix build errors with newer binutils:
../sysdeps/unix/sysv/linux/arm/sigrestorer.S: Assembler messages:
../sysdeps/unix/sysv/linux/arm/sigrestorer.S:30: Error: previous CFI entry not closed (missing .cfi_endproc)
../sysdeps/unix/sysv/linux/arm/sigrestorer.S:31: Error: open CFI at the end of file; missing .cfi_endproc directive

--- a/ports/sysdeps/unix/sysv/linux/arm/sigrestorer.S
+++ b/ports/sysdeps/unix/sysv/linux/arm/sigrestorer.S
@@ -24,10 +24,12 @@
 
 ENTRY(__default_sa_restorer)
 	swi	SYS_ify(sigreturn)
+END(__default_sa_restorer)
 
 #ifdef __NR_rt_sigreturn
 
 ENTRY(__default_rt_sa_restorer)
 	swi	SYS_ify(rt_sigreturn)
+END(__default_rt_sa_restorer)
 
 #endif

